src/Symfony/Component/Console/Output/OutputInterface.php

```diff
@@ -49,13 +49,15 @@ public function writeln($messages, int $options = 0);
 
     /**
      * Sets the verbosity of the output.
+     * 
+     * @param int $level One of self::VERBOSITY_* constants
```

Comment:
Psalm and Phpstan have a [notation for constant enumerations](https://phpstan.org/writing-php-code/phpdoc-types#literals-and-constants):

```suggestion
     * @param self::VERBOSITY_* $level
```

Reply:
Great !